scripts/repos-sync.sh

```
@@ -144,10 +144,111 @@ for repo in $REPOS; do
+            git fetch origin "$main_branch" --quiet
+
+            # Check if we're behind
+            LOCAL=$(git rev-parse HEAD)
+            REMOTE=$(git rev-parse "origin/$main_branch")
+
+            if [[ "$LOCAL" != "$REMOTE" ]]; then
+                if git pull origin "$main_branch" --ff-only 2>/dev/null; then
+                    echo -e "  ${GREEN}✓ Successfully pulled latest changes${NC}"
+
+                    # Show what was updated
+                    git log --oneline HEAD@{1}..HEAD | head -5 | sed 's/^/    /'
```

Shellcheck (0.11.0)

[warning] This { is literal. Check expression (missing ;/\n?) or quote it.

(SC1083)